__tests__/app.test.js

```diff
@@ -55,24 +55,25 @@ describe("GET Endpoints", () => {
     });
   });
 
-  describe("GET /api/articles/ (05)", () => {
+  describe("GET /api/articles/ (05)(11)", () => {
     it("200: Returns object with a single article, selected by id", () => {
       return request(app)
-        .get("/api/articles/2")
+        .get("/api/articles/1")
         .expect(200)
         .then((response) => {
           const { article } = response.body;
           expect(article).toBeInstanceOf(Object);
-          expect(article.article_id).toBe(2);
-          expect(article).toMatchObject({
-            author: expect.any(String),
-            title: expect.any(String),
-            article_id: expect.any(Number),
-            topic: expect.any(String),
-            created_at: expect.any(String),
-            votes: expect.any(Number),
-            article_img_url: expect.any(String),
-          });
+          expect(article.article_id).toBe(1);
+          expect(article.comment_count).toBe('11');
+          expect(article.author).toBe('butter_bridge');
+          expect(article.title).toBe("Living in the shadow of a great man");
+          expect(article.topic).toBe('mitch');
+            expect(article).toMatchObject({
```

Comment:
Rather than refactor a previous test, could we please have a new test for ticket 11? You may need to change this one to toMatchObject or something similar, so it doesn't break!